index.html

```diff
@@ -1 +1,980 @@
+            <p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to
+                pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and
+                irregularities. Those particles were not missing, but would be diluted beyond detection, like spit
+                in the ocean.
+            </p>
+            <p>
+                “SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around
+                it.
+            </p>
+            <p>
+                On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning
+                of the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.
+            </p>
+            <p>
+                Reaching back across 13.8 billion years to the first sliver of cosmic time with telescopes at the
+                South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for
+                Astrophysics detected ripples in the fabric of space-time — so-called gravitational waves — the
+                signature of a universe being wrenched violently apart when it was roughly a trillionth of a
+                trillionth of a trillionth of a second old. They are the long-sought smoking-gun evidence of
+                inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.
+            </p>
+        </section>
+        <section class="sidebar-head">
+            <p>
+                Editor's Picks
+            </p>
+        </section>
+        <section class="section-align1">
+            <div>
+                <p class="tiny-border"></p>
+                <p id="main-mid-p">Unlock more free articles.</p>
+                <a href="" id="bluefont">Create an account or log in</a>
+                <p class="tiny-border1"></p>
+            </div>
+        </section>
+        <section id="section-align2">
+            <aside>
+                <img src="assets/character1.jpg" alt="sidebarimage1">
+                <a href='#'><span>Happened to Those Couples Who Quarantined Together?
+                    </span></a>
+                <img src="assets/character2.jpg" alt="sidebarimage2">
+                <a href='#'><span>‘We Don’t Have to Put Up With This’: A Candid Conversation About Bodies.
+                    </span></a>
+                <img src="assets/character3.jpg" alt="sidebarimage3">
+                <a href='#'><span>How the Trump Era Has Strained, and Strengthened.
+                        reading the
+                        main story
+                    </span></a>
+                <img src="assets/character4.jpg" alt="sidebarimage4">
+                <a href='#'><span>How the Trump Era Has Strained, and Strengthened
+                        reading the
+                        main story
+                    </span></a>
```

Comment:
- [ ]  The `<span>` is not needed [OPTIONAL]

Reply:
Thank you for the above suggestion, we will try to follow from the next project